Persistence/src/main/kotlin/mapper/LineupMapper.kt

```diff
@@ -0,0 +1,35 @@
+package mapper
+
+import dto.LineupDto
+import entities.Lineup
+import java.util.*
+
+internal class LineupMapper : BaseMapper<LineupDto, Lineup> {
+
+    override fun toDto(entity: Lineup): LineupDto {
+        return LineupDto(
+            type = entity.type,
+            position = entity.position,
+            positionNumber = entity.positionNumber
+
+        )
+    }
+    //"id,matchId,teamId,playerId,type,position,positionNumber
```

Comment:
no need for comments, the main reason I usually move away from them is that they are hard to maintain and we can sometimes forget to update them when we update code